redis/files/redis-2.4.conf.jinja

```diff
@@ -106,423 +41,86 @@ save 300 10
 save 60 10000
```

Comment:
Sorry I don't see it. If you do generate other save entries,  don't you do it in the if part from line 34 to 37. save 60 10000 is in the else part(38-42). Isn't it only present if no other snapshot is defined? I think these lines(38-31) are from the default config(http://download.redis.io/redis-stable/redis.conf). I think we can delete the else part if a default isn't wanted, but we should delete it in all config templates. Also we can move this default to map.jinja. But I don't know if it's overwritable then via pillar? But maybe I just didn't get it :)